### PR TITLE
#3021 Adds FAQ update & release notes

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Adds knowledge center content to complete #3021 

## How to Test
Test environment pages:
- [Home (release notes)](https://reitermb.github.io/Test/knowledge-center/)
- [FAQ](https://reitermb.github.io/Test/knowledge-center/faq.html)

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

**Passed a11y scans:**
image
image

### CODE REVIEW

File: product-updates/knowledge-center/index.html [hunk 1]
```diff
@@ -341,6 +341,54 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+                    <li class="usa-icon-list__item" style="margin-top: 16px;">
+            <div class="usa-icon-list__icon text-ink">
+              <img src="../img/added.svg" aria-hidden="true" alt="" style="background-color: #d3ffcc; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                More detailed insights into your files in Submission History
+              </h5>
+              <p>
+              Submission History has been enhanced with new file statuses and more detailed insights about the data in your files. The enhanced data insights are designed to give you a broad understanding of the completeness of your file and provide some insight into the type of errors you can expect to see in the error report if TDP has identified data quality issues. <a href="view-submission-history.html">Read more about Submission History and file statuses.</a>
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]:
```suggestion
              Submission History includes new details about the processing status of your files. These details are designed to give you a broad understanding of the completeness of your file and provide some insight into the type of errors you can expect to see in the error report if TDP has identified data quality issues. <a href="view-submission-history.html">Read more about Submission History and file statuses.</a>
```

**Review by ADPennington — APPROVED** [review]
👍🏾

**Comment by ttran-hub** [comment]
Looks good, a11y 🚀.

**Comment by reitermb** [comment]
@ADPennington Test environment and PR should now be up to date with the latest version of the release note copy (re: Request Access form changes) and the associated screenshot/caption changes in the existing login/new login getting started guides.

File: product-updates/knowledge-center/index.html [hunk 2]
```diff
@@ -341,6 +341,81 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+    <li class="usa-icon-list__item" style="margin-top: 16px;">
+      <div class="usa-icon-list__icon text-ink">
+          <img src="../img/changed.svg" aria-hidden="true" alt="" style="background-color: #b3e2ff; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Improved the usability of TDP's request access form
+        </h5>
+        <p>
+      First-time users to TDP provide their first name, last name, and associated state, tribe, or territory to help TDP administrators confirm access to the system. We've made revisions to the form by adding the jurisdiction type (state, tribe, or territory). After selecting the jurisdiction type, the user can more quickly locate and select the appropriate jurisdiction from the dropdown menu. the eliminate confusion about whether tribes need to select the name of their tribal program or the name of the state it's located in.
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]:
```suggestion
      First-time TDP users provide their first name, last name, and associated state, tribe, or territory. This information helps TDP administrators confirm system access.
      We have updated the form to include a new field for the jurisdiction type (state, tribe, or territory). This change addresses questions from tribal program users about whether to select their tribal program name or the state where the program is located.
      After selecting the jurisdiction type, the user can more quickly locate and select the appropriate jurisdiction from the dropdown menu.
```

**Review by ADPennington — APPROVED** [review]
@reitermb can you spin up a new ticket to update some videos, including the ones to sign up for access, since the request form is changing? i also think STTs may benefit from a video on submission history and error reports. this is otherwise, approved :)

**Comment by reitermb** [comment]
> @reitermb can you spin up a new ticket to update some videos, including the ones to sign up for access, since the request form is changing? i also think STTs may benefit from a video on submission history and error reports. this is otherwise, approved :)

Spun up #3053 #3054. Thanks Alex!

**Comment by reitermb** [comment]
Hi, @lfrohlich. We've got a11y & Alex's content review on this, just let us know if we're good to merge or if you have any other input!

**Review by lfrohlich — APPROVED** [review]
LGTM

**Pull request merged**